Dockerfile

```diff
@@ -20,115 +50,100 @@ RUN apt-get -y update \
     --exclude='freesurfer/lib/qt' && \
     apt-get install -y tcsh bc tar libgomp1 perl-modules curl
 
-# Set up the environment
-ENV OS=Linux \
-    FS_OVERRIDE=0 \
-    FIX_VERTEX_AREA= \
-    SUBJECTS_DIR=/opt/freesurfer/subjects \
-    FSF_OUTPUT_FORMAT=nii.gz \
-    MNI_DIR=/opt/freesurfer/mni \
-    LOCAL_DIR=/opt/freesurfer/local \
-    FREESURFER_HOME=/opt/freesurfer \
-    FSFAST_HOME=/opt/freesurfer/fsfast \
-    MINC_BIN_DIR=/opt/freesurfer/mni/bin \
-    MINC_LIB_DIR=/opt/freesurfer/mni/lib \
-    MNI_DATAPATH=/opt/freesurfer/mni/data \
-    FMRI_ANALYSIS_DIR=/opt/freesurfer/fsfast \
-    PERL5LIB=/opt/freesurfer/mni/lib/perl5/5.8.5 \
-    MNI_PERL5LIB=/opt/freesurfer/mni/lib/perl5/5.8.5 \
-    PATH=/opt/freesurfer/bin:/opt/freesurfer/fsfast/bin:/opt/freesurfer/tktools:/opt/freesurfer/mni/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:$PATH
-
-
-# https://bugs.debian.org/830696 (apt uses gpgv by default in newer releases, rather than gpg)
-RUN set -x \
-	&& apt-get update \
-	&& { \
-		which gpg \
-		|| apt-get install -y --no-install-recommends gnupg \
-	; } \
-# Ubuntu includes "gnupg" (not "gnupg2", but still 2.x), but not dirmngr, and gnupg 2.x requires dirmngr
-# so, if we're not running gnupg 1.x, explicitly install dirmngr too
-	&& { \
-		gpg --version | grep -q '^gpg (GnuPG) 1\.' \
-		|| apt-get install -y --no-install-recommends dirmngr \
-	; } \
-	&& rm -rf /var/lib/apt/lists/*
-
-# apt-key is a bit finicky during "docker build" with gnupg 2.x, so install the repo key the same way debian-archive-keyring does (/etc/apt/trusted.gpg.d)
-# this makes "apt-key list" output prettier too!
-RUN set -x \
-	&& export GNUPGHOME="$(mktemp -d)" \
-	&& gpg --keyserver ha.pool.sks-keyservers.net --recv-keys DD95CC430502E37EF840ACEEA5D32F012649A5A9 \
-	&& gpg --export DD95CC430502E37EF840ACEEA5D32F012649A5A9 > /etc/apt/trusted.gpg.d/neurodebian.gpg \
-	&& rm -rf "$GNUPGHOME" \
-	&& apt-key list | grep neurodebian
-
-RUN { \
-	echo 'deb http://neuro.debian.net/debian trusty main'; \
-	echo 'deb http://neuro.debian.net/debian data main'; \
-	echo '#deb-src http://neuro.debian.net/debian-devel trusty main'; \
-} > /etc/apt/sources.list.d/neurodebian.sources.list
```

Comment:
Why do you want to remove this? It's an important workaround for flaky key server.

Reply:
Obviously I did not understand the purpose of this. Feel free to leave it in!